.docker/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+# Example command:
+#
+#   DOCKER_IMAGE=humble-tutorial docker compose run gpu
```

Comment:
I'd include at least the rolling one here, and the cpu option... and maybe a pointer to the doc here too